File: Python/Program/HDD.py
from HardDrive import HardDrive

class HDD(HardDrive):
    def __init__(self, name="", hbrand="", price=0.0, warranty=0,
                 sbrand="", smodel="", capacity=0, type_="",
                 cache=0, rpm=0):
        super().__init__(name, hbrand, price, warranty, sbrand, smodel, capacity, type_, cache)
        self.rpm = rpm

    def set_rpm(self, rpm):
        self.rpm = rpm

    def get_rpm(self):
        return self.rpm

    def print_info(self):
        print("=== HDD ===")
        super().print_info()
        print(f"RPM           : {self.rpm}")

File: Java/Program/HDD.java
public class HDD extends HardDrive {
    private int rpm;

    public HDD() {}

    public HDD(String brand, String model, int capacity, String type,
               String hwName, String hwBrand, double price, int warranty, int cache,
               int rpm) {
        super(brand, model, capacity, type, hwName, hwBrand, price, warranty, cache);
        this.rpm = rpm;
    }

    public void setRpm(int rpm) { this.rpm = rpm; }
    public int getRpm() { return rpm; }
}
